src/server/foundation/utils.clj

```diff
@@ -0,0 +1,49 @@
+(ns foundation.utils.lacinia
+  (:require [clojure.spec.alpha :as s]))
+
+; maybe use lacinia's own specs
+(comment (s/def ::type (s/or :scalar #{'Int 'String 'ID 'Float 'Boolean '}
```

Comment:
Yes I definitely see some benefits to use spec for those schemas transformations